docs/hardware.rst

```diff
@@ -528,11 +528,11 @@ Previous Server Recommendations
 Intel 5th-gen NUC
 ~~~~~~~~~~~~~~~~~
 
-We previously recommended the
-`NUC5i5MYHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc5i5myhe.html>`__,
-however, it has now reached end-of-life. We will continue to support and
+We previously recommended the NUC5i5MYHE, however,
+it has now reached end-of-life. We will continue to support and
```

Comment:
It might be nice to include a Wayback link here, with the link label "archived copy of specifications" or similar:
https://web.archive.org/web/20190102175756/https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc5i5myhe.html

Reply:
I didn't think of that! Good idea!